chore: sort 'Indexer Management' env variables

rimrakhimov opened the pull request:
Sorting splits all variables in 3 separate groups:
1. In the beginning all variables not having `INDEXER_` prefix are located in their original order
2. Then a group of variables with `INDEXER_DISABLE_` prefix are located sorted alphabetically
3. At the end, all other `INDEXER_` prefixed variables are located sorted alphabetically

vbaranov commented:
We moved the list of supported chains and their Blockscout instances to https://github.com/blockscout/chainscout repository. Please create a PR there, if still relevant.

rimrakhimov committed:
refactor: sort 'Indexer Management' subsection evn variables

    Sorting splits all variables in 3 separate groups:
1. In the beginning all variables not having `INDEXER_` prefix are located in their original order
2. Then a group of variables with `INDEXER_DISABLE_` prefix are located sorted alphabetically
3. At the end, all other `INDEXER_` prefixed variables are located sorted alphabetically
